pom.xml

```diff
@@ -90,17 +90,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-    <!--<profiles>-->
-        <!--<profile>-->
-            <!--<id>dev</id>-->
-            <!--<dependencies>-->
-                <!--<dependency>-->
-                    <!--<groupId>net.smartcosmos</groupId>-->
-                    <!--<artifactId>smartcosmos-ext-relationships-rdao</artifactId>-->
-                    <!--<version>3.0.0-SNAPSHOT</version>-->
-                    <!--<scope>runtime</scope>-->
-                <!--</dependency>-->
-            <!--</dependencies>-->
-        <!--</profile>-->
-    <!--</profiles>-->
+    <profiles>
```

Comment:
With the package projects this should not be necessary.  It should be possible to run from the package project with no profiles necessary.